front-end/src/components/CreateTodo.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {} from "react";
+import { TodoType } from "../types";
+import useCreateTodo from "../hooks/useCreateTodo";
+
+function CreateTodo() {
+    const AddHandler = useCreateTodo()
+    const todo_item: TodoType = {
```

Comment:
`todoItem` should be inside the hook.

Reply:
todoItem fix hojayega, Mujhe wo krne ka man nhi tha, doesn't look good, this could be considered as replacement for what would be a form